### PR TITLE
Bender: Bump `axi_rt` AXI real-time unit

### PR DESCRIPTION
**Pull request by alex96295**
- [x] Bump cheshire: point to standalone, open-source AXI RT
- [x] Bump common_cells to `v1.31.1`

### CODE REVIEW

**Comment by alex96295** [comment]
@thommythomaso rfc

File: carfield.mk [hunk 1]
```diff
@@ -30,6 +30,10 @@ PLIC_NUM_INTRS := 89
 # Serial Link configuration in cheshire
 SERIAL_LINK_NUM_BITS := 16
 
+# AXI Real-Time unit configuration in Carfield
+AXIRT_NUM_MGRS := 10
+AXIRT_NUM_SUBS := 13
```
Comment (thommythomaso) [inline carfield.mk:35]: this should be 2 if I am not mistaken.

Comment (alex96295) [reply carfield.mk]: done

**Comment by alex96295** [comment]
Rebased onto main

**Pull request merged**